.github/workflows/docker-ros.yml

```diff
@@ -4,27 +4,57 @@ on: push
 
 jobs:
 
-  ros:
+  ros-noetic:
     runs-on: ubuntu-latest
     steps:
       - uses: ika-rwth-aachen/docker-ros@main
         with:
+          platform: amd64,arm64
+          target: dev,run
           image-tag: ros
           base-image: rwthika/ros:noetic
           command: roslaunch etsi_its_conversion converter.ros1.launch
-          platform: amd64,arm64
-          target: dev,run
+          enable-industrial-ci: 'true'
           enable-recursive-vcs-import: 'false'
 
-  ros2:
+  ros2-humble:
     runs-on: ubuntu-latest
     steps:
       - uses: ika-rwth-aachen/docker-ros@main
         with:
+          platform: amd64,arm64
+          target: dev,run
           image-tag: ros2
           base-image: rwthika/ros2:humble
           command: ros2 launch etsi_its_conversion converter.launch.py
+          enable-industrial-ci: 'true'
+          enable-push-as-latest: 'true'
```

Comment:
Only set `enable-push-as-latest` true for one configuration. I would suggest ros2-jazzy.